adafruit_circuitplayground/express.py

```diff
@@ -61,17 +75,7 @@ def __init__(self):
           while True:
               pass
         """
-
-        # Define sensors:
-        self._temp = adafruit_thermistor.Thermistor(board.TEMPERATURE, 10000, 10000, 25, 3950)
-        self._light = Photocell(board.LIGHT)
-
-        # Define audio:
-        self._speaker_enable = digitalio.DigitalInOut(board.SPEAKER_ENABLE)
-        self._speaker_enable.switch_to_output(value=False)
-
-        self.sample = None
-        self.sine_wave = None
+        return pixels
```

Comment:
This seems wrong to me. Shouldn't it be `self._pixels`?

Reply:
Yes it should be. Resolved and tested.